superset/security/manager.py

```diff
@@ -1956,11 +1957,14 @@ def raise_for_access(
                 return
 
             if query:
+                # make sure the quuery is valid SQL by rendering any Jinja
```

Comment:
```suggestion
                # make sure the query is valid SQL by rendering any Jinja
```

Reply:
Good catch! Let me fix this in another PR so we can merge this already.